src/components/landing-page/HeroSection.tsx

```diff
@@ -2,21 +2,23 @@ import React from "react";
 import Link from "next/link";
 import Image from "next/image";
 
-import { Wrapper } from ".";
 import HeroImageDestop from "/public/images/hero-desktop-image.webp";
-import HeroImageMobile from "/public/images/hero-mobile-image.webp";
 import { ArrowLinkUpRight } from "@bitcoin-dev-project/bdp-ui/icons";
+import circleBackground from "/public/svgs/circle-background.svg";
+import circleBackgroundMobile from "/public/svgs/circle-background-mobile.svg";
+import mobileDesktopImage from "/public/svgs/mobile-desktop-image.svg";
+import Wrapper from "../layout/Wrapper";
 
 const HeroSection = () => {
   return (
-    <Wrapper className='flex items-center justify-between min-h-[calc(100vh-141px)] pr-0 max-md:flex-col max-md:items-start max-md:justify-normal max-md:gap-6 max-md:pt-14 max-md:min-h-fit max-md:px-0'>
+    <Wrapper className='flex items-center justify-between min-h-[calc(100vh-141px)] pr-0 max-md:flex-col max-md:items-start max-md:justify-normal max-md:gap-6 max-md:pt-14 max-md:min-h-fit max-md:px-0 z-10 relative overflow-hidden'>
       <section className='w-[40%] flex flex-col gap-10 max-md:w-full max-md:gap-6 max-md:px-4'>
         <section className='text-black flex flex-col gap-6 max-md:max-w-[368px]'>
           <h1 className='2xl:text-[72px] text-[64px] leading-[130%] max-xl:text-5xl max-xl:leading-[130%] max-[953px]:text-[38px] max-[953px]:leading-[130%] max-md:text-[42px]'>
             Unlock the treasure trove of technical bitcoin transcripts
           </h1>
           <p className='text-xl max-lg:text-lg max-md:text-base'>
-            <span className='font-bold'>1042+ transcripts</span> growing every day. Thanks to people <br /> like you.
+            <span className='font-bold'>1042+ transcripts</span> growing every day. <br /> Thanks to people like you.
```

Comment:
Now the second sentence is always at a separate line. I think that we can use `white-space: nowrap;` for the second sentence so that the entire sentence or block of text stays together and doesn't break between lines.

Another option is to use non-breaking spaces (`&nbsp;`) between the words of the second sentence.

Reply:
I don't think the second sentence breaks on any screen size, but we can add a text-nowrap to the entire sentence to ensure the block stays together.
The text only breaks from 803px - 768px.

Reply:
I think we should move all conversations to the production pr, so these can be tracked

Reply:
> I don't think the second sentence breaks on any screen size, but we can add a text-nowrap to the entire sentence to ensure the block stays together. The text only breaks from 803px - 768px.

You have the `<br />` there so the second sentence is always on a new line.